site/source/docs/api_reference/module.rst

```diff
@@ -119,7 +119,7 @@ The following ``Module`` attributes affect code execution. Set them to customize
 
 .. js:attribute:: Module.preinitializedWebGLContext
 
-  If building with -s GL_PREINITIALIZED_CONTEXT=1 set, you can set ``Module.preinitializedWebGLContext`` to a precreated instance of a WebGL context, which will be used later when initializing WebGL in C/C++ side. Precreating the GL context is useful if doing GL side loading (shader compilation, texture loading etc.) parallel to other page startup actions, and/or for detecting WebGL feature support, such as GL version or compressed texture support up front on a page before or in parallel to loading up any compiled code.
+  If building with ``-s GL_PREINITIALIZED_CONTEXT=1`` set, you can set ``Module.preinitializedWebGLContext`` to a precreated instance of a WebGL context, which will be used later when initializing WebGL in C/C++ side. Precreating the GL context is useful if doing GL side loading (shader compilation, texture loading etc.) parallel to other page startup actions, and/or for detecting WebGL feature support, such as GL version or compressed texture support up front on a page before or in parallel to loading up any compiled code.
```

Comment:
While you are there can you drop the `=1` here?

Reply:
oh, my fault, i'll remove it.

Reply:
i've just proofreading the syntax of doc, i havent use WebGL in my project, so that i've no experience about this option

Reply:
No worries.  All options are implicitly `=1` these days so `=1` is always redundant.